# Dimensions of the load's bounding box in meters
#
# Part of the state message

# Absolute length of the load's bounding box in meters
#
# Field type: Required
float64 length

# Absolute width of the load's bounding box in meters
#
# Field type: Required
float64 width

# Absolute height of the load's bounding box in meters
#
# Field type: Optional
float64[<=1] height
